doc/useful_ros_tools/rqt.rst

```diff
@@ -0,0 +1,31 @@
+.. _rqt-label:
+
+RQT
+===
+`RQT <https://wiki.ros.org/rqt>`_ is a Qt-based framework for GUI development in ROS.
+All GUI applications from ROS are written using RQT, for example, ``rviz`` and ``ros_bag``.
+The GUI applications are written as RQT plugins, which can be included in windows, so one
+window can include multiple RQT plugins. For example, if you run
+
+.. code::
+
+    rqt  # a roscore has to be running
+
+You can select multiple plugins from the ``Plugins`` menu in the top left. Once you have
+selected a few plugins you can save the current 'perspective'. A perspective is a combination
+of RQT plugins in a defined configuration. In the ``Perspectives`` menu you can save, import,
+export and set perspectives as default. This is useful if you want, for example, to have the
+input device and the gait selection tool next to each other in one window.
+
+.. figure:: images/rqt_perspective.png
+   :align: center
+
+   Example rqt perspective
+
+The :monitor:`project-march/monitor <>` repository contains all the rqt plugins used for
+the |march|.
```

Comment:
To be formally correct, the gait_generator is a RQT plugin as well, right?

Reply:
Yes, forgot to mention that one